index.html

```diff
@@ -15,16 +15,19 @@
            <nav class="navItems">
             <a class="logo" href="">Welcome</a>
             <a class="bars"> <i class="fa-solid fa-bars"></i> </a>
-           </nav>
```

Comment:
- [OPTIONAL] Kindly make this be on the same line 

![0111111111](https://user-images.githubusercontent.com/79058364/161378280-f592990e-f87d-4d57-a492-168f16a1d0c3.png)